app/src/main/java/com/sunasterisk/music_72/screen/fragment/home/HomeFragment.kt

```diff
@@ -0,0 +1,61 @@
+package com.sunasterisk.music_72.screen.fragment.home
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.sunasterisk.music_72.R
+import com.sunasterisk.music_72.data.anotation.GenreName
+import com.sunasterisk.music_72.data.model.Genre
+import com.sunasterisk.music_72.data.source.local.TrackLocalDataSource
+import com.sunasterisk.music_72.data.source.remote.TrackRemoteDataSource
+import com.sunasterisk.music_72.data.source.remote.connection.RetrofitClient
+import com.sunasterisk.music_72.data.source.repository.TrackRepositoryImplementor
+import com.sunasterisk.music_72.databinding.FragmentHomeBinding
+import com.sunasterisk.music_72.screen.MainActivity
+import com.sunasterisk.music_72.screen.factory.ViewModelFactory
+import com.sunasterisk.music_72.utils.MusicUtils
+import com.sunasterisk.music_72.utils.setupToolbar
+import kotlinx.android.synthetic.main.fragment_home.*
+
+class HomeFragment : Fragment() {
+    private lateinit var homeBinding: FragmentHomeBinding
+    private lateinit var viewModel: HomeViewModel
+    private lateinit var factory: ViewModelFactory
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        activity?.let { factory = ViewModelFactory.getInstance() }
+        homeBinding = FragmentHomeBinding.inflate(inflater, container, false)
```

Comment:
dùng data binding sai
```suggestion
        homeBinding = DataBindingUtil.inflate(inflater, "id layout view", container, false)
```